mobs.lua

```diff
@@ -0,0 +1,9 @@
+-- mobs and beowolf don't play nicely together when stone is made
+-- dangerous by beowolf, we counteract that here to make mobs not fear
```

Comment:
```suggestion
-- mobs and beowulf don't play nicely together when stone is made
-- dangerous by beowulf, we counteract that here to make mobs not fear
```
Source: https://en.wikipedia.org/wiki/Beowulf

Reply:
thanks :)